telegram/ext/basepersistence.py

```diff
@@ -146,7 +146,18 @@ def replace_bot(cls, obj: object) -> object:
         if isinstance(obj, (list, tuple, set, frozenset)):
             return obj.__class__(cls.replace_bot(item) for item in obj)
 
-        new_obj = copy(obj)
+        try:
+            new_obj = copy(obj)
+        except Exception:
+            warnings.warn(
+                'BasePersistence.replace_bot caught an error while trying to copy an object. '
+                'Objects that can not be copied will be assumed to not contain a telegram.Bot '
+                'instance and will not be handled further. See the docs of '
+                'BasePersistence.replace_bot for more information.',
+                RuntimeWarning,
```

Comment:
Try to make this text shorter.
People will get bored and look away with such a long text...